data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/EventMatcher.java

```diff
@@ -101,6 +107,10 @@ public boolean match(final CloudEvent event) {
     return stringProvider.apply(s);
   }
 
+  private static boolean isNotAny(final String value) {
```

Comment:
maybe `notEmpty`?

Reply:
Yeah, I was using the same terminology of the docs, but not a strong opinion here.

Reply:
Done: https://github.com/knative-sandbox/eventing-kafka-broker/pull/48/commits/54a3b82bf87698bc7c3b676640a64d97789bbd67